### ISSUE TITLE
Accessibility Improvements

### ISSUE THREAD

**Issue by Tarun-B-12** [VarshikaReddy/SE#7]
Users with disabilities may not be able to access the LMS completely. Lack of alt text, difficulty using the keyboard, or compatibility with screen readers are all possible concerns.
